pandas/core/config_init.py

```diff
@@ -514,6 +515,13 @@ def use_inf_as_na_cb(key):
         validator=str,
     )
 
+with cf.config_prefix("io.excel.xlsb"):
+    cf.register_option(
+        "reader",
+        "auto",
+        reader_engine_doc.format(ext="xlsb", others=", ".join(_ods_options)),
```

Comment:
this joins on the ods_options? is that right?

Reply:
Hmm I do wonder about that sometimes